src/containers/Footer/index.js

```diff
@@ -17,16 +21,24 @@ import flagBrazil from '../shared/images/flag_brazil.png';
 import flagMexico from '../shared/images/flag_mexico.png';
 import './footer.css';
 
-const LANGUAGES = {
-  'en-US': { title: 'English', flag: flagUSA, code: 'en-US' },
-  'zh-Hans': { title: '中文', flag: flagChina, code: 'zh-Hans' },
-  'ja-JP': { title: '日本語', flag: flagJapan, code: 'ja-JP' },
-  'ko-KP': { title: '한국어', flag: flagKorea, code: 'ko-KP' },
-  'es-MX': { title: 'Español', flag: flagMexico, code: 'es-MX' },
-  'pt-BR': { title: 'Português', flag: flagBrazil, code: 'pt-BR' },
-};
+const LANGUAGE_ORDER = [
+  { title: 'English', flag: flagUSA, code: 'en-US' },
+  { title: '中文', flag: flagChina, code: 'zh-Hans' },
+  { title: '日本語', flag: flagJapan, code: 'ja-JP' },
+  { title: '한국어', flag: flagKorea, code: 'ko-KP' },
+  { title: 'Español', flag: flagMexico, code: 'es-MX' },
+  { title: 'Português', flag: flagBrazil, code: 'pt-BR' },
+];
```

Comment:
combines `LANGUAGES` and `ORDER` into one const

Reply:
Does this change how translations are done at all? Is the library we have doing the translations affected at all by this change?

Reply:
nope - this is only for this file. you can see how this is used from the changes i made in this file. it should have been written as an ordered array of objects in the first place